src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/BookBinder.java

```diff
@@ -97,6 +106,20 @@ private boolean isCompatible(@Nullable ItemStack item) {
         return item != null && item.getType() == Material.ENCHANTED_BOOK;
     }
 
+    private boolean hasIllegalEnchants(@Nullable Map<Enchantment, Integer> enchantments) {
+        if (enchantments == null) {
+            return false;
+        }
+
+        for (Map.Entry<Enchantment, Integer> entry : enchantments.entrySet()) {
+            int maxLevel = bypassVanillaMaxLevel.getValue() ? customMaxLevel.getValue() : entry.getKey().getMaxLevel();
```

Comment:
Don't want to use a ternary here (See: [Code Style](https://github.com/Slimefun/Slimefun4/blob/master/CONTRIBUTING.md#7-style-preferences))

Reply:
```java
if (bypassVanillaMaxLevel.getValue() && entry.getValue() > customMaxLevel.getValue()) {
    return true;
} else if (entry.getValue() > entry.getKey().getMaxLevel()) {
    return true;
}
```
Would this be better?

Reply:
or in 1 if better?
```java
if (bypassVanillaMaxLevel.getValue() && entry.getValue() > customMaxLevel.getValue() || !bypassVanillaMaxLevel.getValue() && entry.getValue() > entry.getKey().getMaxLevel()) {
    return true;
}
```

Reply:
second would be better

Reply:
Fixed